Add missing information in the smart-contract-architecture and in summary (#468)
keep-starknet-strange/satoru: book/src/smart-contracts-architecture/withdrawal-module.md

# Withdrawal Module

> The Withdrawal Module is devised to manage the operations related to withdrawals, ensuring secure and efficient handling of funds within the system.

Reviewer: ```suggestion
The Withdrawal Module role is to manage the operations related to withdrawals.
```
